back-end/cmp/settings/development.py

```diff
@@ -12,3 +12,16 @@
         "PORT": cmp_db_port,
     }
 }
+
+
+# celery settings
+# CELERY_BROKER_URL = "amqp://localhost"
+# CELERY_RESULT_BACKEND = "amqp://localhost"
+
+# email settings
+EMAIL_HOST = os.getenv("EMAIL_HOST")
```

Comment:
add these to the `base.py` file instead. They will both be required in production and development for now

Reply:
resolved